tree/ntuple/v7/inc/ROOT/RPageStorage.hxx

```diff
@@ -45,13 +45,17 @@
 #include <vector>
 
 namespace ROOT {
+
+namespace Internal {
+class RNTupleCompressor;
+}
```

Comment:
Actually I think this can be entirely removed after 497da18691faa3b46374264206f74cd1308f50a2...